src/policy/fees.h

```diff
@@ -180,13 +181,10 @@ class CBlockPolicyEstimator : public CValidationInterface
     static constexpr double SUFFICIENT_TXS_SHORT = 0.5;
 
     /** Minimum and Maximum values for tracking feerates
-     * The MIN_BUCKET_FEERATE should just be set to the lowest reasonable feerate we
-     * might ever want to track.  Historically this has been 1000 since it was
-     * inheriting DEFAULT_MIN_RELAY_TX_FEE and changing it is disruptive as it
-     * invalidates old estimates files. So leave it at 1000 unless it becomes
```

Comment:
I am not sure if we need a release note, since according to this note, it will invalidate old estimates files? Maybe someone else can chime in on this

Reply:
Done